### PR TITLE
Ab gallery

### PR DESCRIPTION
**Pull request by djbrownbear**
updated gallery page with photos from Amanda's IG

### CODE REVIEW

File: gallery.html
Comment (djbrownbear) [inline gallery.html:1]: not sure if the paragraph on line 43 is still necessary, but wanted to see what it looks like for now...

Comment (djbrownbear) [reply gallery.html:1]: also, the layout of the photos section was borrowed from elements.html

**Review by Yu-HuanWu — COMMENTED** [review]
when fixing merge conflict, github will show you something like

```
<<<<<<< AB-gallery
										<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
										<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
										<li><a href="#" class="icon brands fa-snapchat-ghost"><span class="label">Snapchat</span></a></li>
										<li><a href="https://instagram.com/mandyb.bakes" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
										<li><a href="#" class="icon brands fa-medium-m"><span class="label">Medium</span></a></li>
=======
										<!-- <li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li> -->
										<!-- <li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li> -->
										<!-- <li><a href="#" class="icon brands fa-snapchat-ghost"><span class="label">Snapchat</span></a></li> -->
										<li><a href="https://instagram.com/mandyb.bakes" class="icon brands fa-instagram"><span
													class="label">Instagram</span></a></li>
										<!-- <li><a href="#" class="icon brands fa-medium-m"><span class="label">Medium</span></a></li> -->
>>>>>>> main
```

so basically you just have to delete the one you don't want to keep, in this case everything from <<<<<<< AB-gallery to ========
and don't forget to delete >>>>>>> main as well. the <<<<, =====, and >>>>> are just github's way of differentiating their codes from the rest of the codes.

File: about.html [hunk 1]
```diff
@@ -22,12 +22,12 @@
 
 							<!-- Header -->
 								<header id="header">
-									<a href="index.html" class="logo"><strong>Editorial</strong> by HTML5 UP</a>
+									<a href="index.html" class="logo"><strong>Mandy B. Bakes</strong></a>
 									<ul class="icons">
 										<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
 										<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
```
Comment (Yu-HuanWu) [inline about.html:28]: weird, on the main branch this should be commented out, i wonder why it's not commented out here.

**Review by Yu-HuanWu — APPROVED** [review]

**Pull request merged**